File: Week 1/Problem Set 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //input for the height
    int n;
    do
    {
        printf("Height: ");
        scanf("%i", &n);
    }
    while (n < 1 || n > 8);

    //print pyramids
    for (int i = 1; i <= n; i++)
    {
        //print first whitespaces
        for (int j = 0; j < n - i; j++)
        {
            printf(" ");
        }
        //prints first pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        //print separator
        printf("  ");
        //print secod pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        //print new line for the next row of the pyramids
        printf("\n");
    }
}

File: Week 6/Problem Set 6/sentimental-mario-more/mario.py
while True:
    try:
        n = int(input("Height: "))
        if n > 8 or n < 1:
            raise ValueError
    except ValueError:
        continue
    else:
        break
for i in range(1, n + 1):
    print(" " * (n - i), end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)
